lib/context/User/User.context.tsx

```
@@ -0,0 +1,670 @@
+      return total + (parseFloat(item.price as string) || 0) * item.quantity;
+    }, 0).toFixed(2);
+  }, [cart]);
+
+  return (
+    <UserContext.Provider
+      value={{
+        isLoggedIn: !!token,
+        loadingProfile: loadingProfile && calledProfile,
+        errorProfile,
+        profile:
+          dataProfile && dataProfile.profile ? dataProfile.profile : null,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)